annotationframeworkclient/chunkedgraph.py

```diff
@@ -467,10 +473,70 @@ def get_lineage_graph(self, root_id, timestamp_past=None, timestamp_future=None)
 
         url = self._endpoints["handle_lineage_graph"].format_map(endpoint_mapping)
         r = handle_response(self.session.get(url, params=data))
-        return r
+
+        if as_nx_graph:
```

Comment:
i don't think this one line difference is worth adding the dependency to this package personally.

Reply:
I see you are gonna use it to make client side connivence functions, i commented before reading the whole thing.